cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -211,8 +211,8 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 
 	hostOS, err := common.GetRedHatRelease()
 	if err != nil {
-		osbuildJobResult.JobError = clienterrors.WorkerClientError(clienterrors.ErrorBuildJob, err.Error())
-		return nil
+		logrus.Warn(err.Error())
+		hostOS = "(unknown)"
```

Comment:
nitpick: I don't think that it is necessary to put the "unknown" into brackets, but not harmful either.

Reply:
Just trying to make sure there's less of a chance it gets interpreted as saying we're running on a distro called _Unknown_ :laughing:

Reply:
Fair point, one can never know 🙄

Reply:
Is a fallback defined in the os-release spec? I seem to remember it being `linux`.

Reply:
`If not set, a default of "ID=linux" may be used.` from https://www.freedesktop.org/software/systemd/man/os-release.html#ID=